biit_server/http_responses.py

```diff
@@ -10,3 +13,18 @@ def http200(description: str = ""):
     if description == "":
         return "OK", 200
     return f"OK: {description}", 200
+
+
+def jsonHttp200(message: str, data):
+    """Http 200 response with json as data
+
+    Args:
+        message (str): Message to be sent
+        data: Data to be sent 
+
+    Returns:
+        str: Json combination of data and message
+    """
+    json = list(data)
```

Comment:
i think this can be replaced by forcing the data parameter to be type hinted as a `dict`

Reply:
Type hinting doesn't force the data being sent in as a specific type, so this must be kept